real_estate/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "category": "real estate",
+    "description": "Documents containing real estate information",
```

Comment:
Once this is in the closing disclosures folder:

`description` should be "A statement of the finalized mortgage loan terms and closing costs"
`display_name` should be "Closing disclosures"

Reply:
@ethulia just did that and pushed